pages.es/common/difft.md

```diff
@@ -0,0 +1,33 @@
+# difft
+
+> Compara archivos o directorios basándose en la sintaxis del lenguaje de programación.
+> Vea también: `delta`, `diff`.
+> Más información: <https://difftastic.wilfred.me.uk/introduction.html>.
+
+- Compara dos archivos o directorios:
+
+`difft {{ruta/al/archivo_o_directorio1}} {{ruta/al/archivo_o_directorio2}}`
+
+- Informa únicamente la presencia de diferencias entre los archivos:
+
+`difft --check-only {{ruta/al/archivo1}} {{ruta/al/archivo2}}`
+
+- Especifica el modo de visualización (por defecto es `side-by-side`):
+
+`difft --display {{side-by-side|side-by-side-show-both|inline|json}} {{ruta/al/archivo1}} {{ruta/al/archivo2}}`
+
+- Ignora comentarios al comparar:
+
+`difft --ignore-comments {{ruta/al/archivo1}} {{ruta/al/archivo2}}`
+
+- Activa/desactiva el resaltado sintáctico del código fuente (por defecto está activado):
```

Comment:
```suggestion
- Activa o desactiva el resaltado sintáctico del código fuente (por defecto está, activado):
```

Reply:
please fix that lasting `,`

Reply:
Ah, meant to put it after `por defecto`. How's that?

Reply:
without `,` is perfect